aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <artifactId>spring-cloud-azure-starter-active-directory-resource-server-by-filter-stateless</artifactId>
   <version>1.0.0</version>
-
+  <packaging>jar</packaging>
```

Comment:
Why this line is necessary?

Reply:
just follow the same pattern with other samples